include/RAJA/index/IndexValue.hpp

```diff
@@ -334,26 +334,30 @@ constexpr RAJA_HOST_DEVICE RAJA_INLINE
   return val;
 }
 
-namespace internal{
-template<typename FROM, typename Enable = void>
-struct StripIndexTypeT {
-    using type = FROM;
+namespace internal
+{
```

Comment:
It would be nice if it inserted a blank line between namespace declarations and the contents inside.